src/api/swagger.json

```diff
@@ -15,10 +15,8 @@
     ],
```

Comment:
Findings overall in swagger file:
- Entity for user is wrong (e.g. password in POST, updatedAt...)
- Entity for pets is wrong (e.g. userId is string, updatedAt...)
- GET,PUT,DELETE /users/{id} is wrong as it should be a string in path
- GET,PUT,DELETE /pets/{id} is wrong as it should be a string in path
- Check response types as they don't match sometimes (e.g. POST /pets)
- DELETE /users/{id} and /pets/{id} results in 404 but the entity is deleted